src/runtime/contrib/dnnl/dnnl_json_runtime.cc

```diff
@@ -1041,6 +1056,70 @@ class DNNLJSONRuntime : public JSONRuntimeBase {
                          {DNNL_ARG_DST, dst_memory}});
   }
 
+  void DensePack(const size_t& nid) {
```

Comment:
`DensePack` is almost identical to the original `Dense` function. Could you apply your change directly to the `Dense` function to eliminate this duplication? Only two lines including the definition of `weight_layout` and `weight_md` needs to be modified. And we get bf16 support in this way.

Reply:
Hi @yangulei , Thanks for the comments. I addressed this issue on latest patch.